doc/source/design/board/gd32vf103c_longan_nano.rst

```diff
@@ -79,7 +95,7 @@ are fixed to ``flashxip`` and ``n205``.
 * There are two versions of this board avail. One with 64k flash and one with 128k flash.
 
   - You have to change the linker script to set the right flash size. By default, 128k flash is set.
-  
+
 * You can find its linker script in ``SoC/gd32vf103/Board/gd32vf103c_longan_nano/Source/GCC/``
 
   - ``gcc_gd32vf103_flashxip.ld``: Linker script file for ``DOWNLOAD=flashxip``
```

Comment:
The linker script file are different for different version for PR #18 , could you also add some notes here.

Reply:
I think if you can merge this PR, than @tuupola can add the description of his work. Would this be ok?

Reply:
That is ok for me.  I will merge it now.